src/components/ProgressBar/ProgressBar.story.js

```diff
@@ -33,4 +34,28 @@ storiesOf(`${GROUPS.COMPONENTS}|ProgressBar`, module)
         </ProgressBar>
       </div>
     ))
+  )
+  .add(
+    'ProgressBar with knobs',
+    withInfo()(() => {
+      const size = select(
+        'Size',
+        {
+          kilo: ProgressBar.KILO,
+          mega: ProgressBar.MEGA,
+          giga: ProgressBar.GIGA
+        },
+        ProgressBar.KILO
+      );
+      const max = number('Maximum number', 10);
+      const value = number('Value', 5);
+      const children = text('Text value', `${value}/${max}`);
+      return (
+        <div style={{ width: '25vw' }}>
+          <ProgressBar value={value} max={max} size={size}>
+            {children}
+          </ProgressBar>
+        </div>
+      );
+    })
```

Comment:
Thanks for adding these knobs! Since there's no need for them anymore, can you please remove the other stories and make this one the default one?